MAUI/CheckBox/Events.md

```diff
@@ -55,4 +55,38 @@ This event occurs when the value or state of the [`IsChecked`](https://help.sync
 {% endhighlight %}
 {% endtabs %}
 
-![.NET MAUI CheckBox](Images/Getting-Started/tristate.png)
\ No newline at end of file
+![.NET MAUI CheckBox](Images/Getting-Started/tristate.png)
+
+## StateChanging event
+
+The StateChanging event is triggered when the CheckBox control is tapped, indicating that the state of the [`IsChecked`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Buttons.SfCheckBox.html#Syncfusion_Maui_Buttons_SfCheckBox_IsChecked) property in the Checkbox control is about to change. The StateChangingEventArgs provides the following properties:
```

Comment:
Highlight the event and class names like below,

The `StateChanging` event is triggered when the .NET MAUI CheckBox control is tapped, indicating that the state of the [`IsChecked`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Buttons.SfCheckBox.html#Syncfusion_Maui_Buttons_SfCheckBox_IsChecked) property in the Checkbox control is about to change. The `StateChangingEventArgs` provides the following properties:

Reply:
Highlighted the event and class name.